pio/lib/Sender/Sender.cpp

```diff
@@ -485,3 +487,237 @@ bool SenderClass::sendTCONTROL(String server, uint16_t port)
     stopclient();
     return true;
 }
+
+//Blynk HTTP was taking 2 seconds longer and did not show in the App
+//when device was connected, therefore best to use their API.
+bool SenderClass::sendBlynk(char* token)
+{
+    serializeJson(_doc, Serial);
+
+    Blynk.config(token);
+
+    byte i = 0;
+    int _pin = 0;
+    String _value;
+
+    while (!Blynk.connected() && i<100)
+    {
+      Blynk.run();
+      i++;
+      delay(50);
+    }
+        
+    if (Blynk.connected())
+    {
+        CONSOLELN(F("\nConnected to the Blynk server, sending data"));
+
+        for (const auto &kv : _doc.as<JsonObject>())
+        {
+            _pin = atoi(kv.key().c_str());
+            _value = kv.value().as<String>();
+            Blynk.virtualWrite(_pin, _value);
+        }
+    }
+
+    else {
+        CONSOLELN(F("\nFailed to connect to Blynk, going to sleep"));
+        return false;
+    }
+    
+    delay(300);     //delay to allow last value to be sent;
+    return true;
+}
+
+bool SenderClass::sendBlynkHTTP(String token)
```

Comment:
If this http method is not used, can we remove the dead code?

Reply:
ok